Sources/PostgresNIO/New/PSQLPreparedStatement.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  File.swift
+//  
+//
+//  Created by Fabian Fett on 25.01.21.
+//
+
+struct PSQLPreparedStatement {
```

Comment:
Ditch Xcode template headers.
```suggestion
struct PSQLPreparedStatement {
```